### PR TITLE
docs(section): add search functionality

### PR DESCRIPTION
**Pull request by santiagoaloi**

### CODE REVIEW

File: packages/docs/src/components/api/Section.vue [hunk 1]
```diff
@@ -1,46 +1,42 @@
...
-        @input="filter = $event"
+        prepend-inner-icon="mdi-magnify"
+        single-line
+        hide-details
       />
-    </div> -->
+
+    </div>
+
+    <!-- TODO: investigate what this component does, if it even exists -->
```
Comment (KaelWD) [inline packages/docs/src/components/api/Section.vue:17]: Enable inline API in settings then go to a component page.

File: packages/docs/src/components/api/Section.vue [hunk 2]
```diff
@@ -51,15 +47,48 @@
...
+    const component = sectionMap[props.section as keyof typeof sectionMap]
+    return component || null
+  })
+
+  // 👉 Seection Search logic
+  const search = ref()
+  const searchableFields = ['name', 'description']
+
+  const filteredItems = computed(() => {
+    return items.value.filter(item =>
```
Comment (KaelWD) [inline packages/docs/src/components/api/Section.vue:78]: `(item: any) =>` or define a proper interface for `items`

**Comment by KaelWD** [comment]
The v2 inline API had a single search for all item types, it was only separate on API pages.
Screenshot_20230331_224711

**Comment by s-rigaud** [comment]
@KaelWD I think this PR might be closed as this feature as recently been implemented 😊

**Pull request closed without merging**